src/components/PageBanner/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './PageBanner.css';
```

Comment:
this component needs a visible prop. visible = true or false which causes the pagebanner to show